scripts/datacommons/place_cohorts/intl/place_page_compare_countries.mcf

```diff
@@ -0,0 +1,66 @@
+
+Node: country/USA
+dcid: "country/USA"
+typeOf: schema:Place
```

Comment:
why not typeOf Country?

Reply:
The code is general, to enforce best practice of e.g. not causing SF being labeled as City and County. We were going to/already added constraints to only allow typeOf from trusted provenances, but this is still better practice if the point is not to add new typeOf.